package.json

```diff
@@ -50,6 +50,7 @@
     "object-hash": "^2.0.3",
     "sns-validator": "^0.3.4",
     "socket.io": "^2.3.0",
+    "socket.io-redis": "^6.0.1",
```

Comment:
This is an adapter which will allow us to scale up the API in production without having to worry about clients connected to different API instances.

See: https://socket.io/docs/v3/using-multiple-nodes/index.html